### PR TITLE
Add SQLi in CodeIgniter

### PR DESCRIPTION
**Pull request by kenjis**
The validator says:
> Invalid branch name "3.0-stable"

But I don't know how to do.

### CODE REVIEW

File: codeigniter/framework/2016-07-26-1.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: Critical SQL injection bug in the ODBC database driver
+link: http://forum.codeigniter.com/thread-65803.html
+cve: ~
+branches:
+    3.0-stable:
```
Comment (fabpot) [inline codeigniter/framework/2016-07-26-1.yaml]: should be just "3.0"

File: codeigniter/framework/2016-07-26-1.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        time:     2015-07-26 19:42:05
+        versions: ['<=3.0.6']
```
Comment (fabpot) [inline codeigniter/framework/2016-07-26-1.yaml]: Are version 1 and 2 also vulnerable?

**Comment by fabpot** [comment]
Any news on this one?

File: codeigniter/framework/2016-07-26-1.yaml (hunk 2 above)
Comment (kenjis) [inline codeigniter/framework/2016-07-26-1.yaml]: Probably yes, but nobody can install version 1 and 2 via Composer.

File: codeigniter/framework/2016-07-26-1.yaml (hunk 1 above)
Comment (kenjis) [inline codeigniter/framework/2016-07-26-1.yaml]: I will fix it.

**Pull request merged**